src/components/error-page.js

```diff
@@ -29,7 +29,7 @@ function ErrorPage({ error, settings, uiLanguage, i18n }) {
       </main>
       <footer className="ErrorPage-footer">
         <div className="ErrorPage-footerContent">
-          {(settings.footerLinks?.[uiLanguage] || []).map((fl, index) => (
+          {(settings?.footerLinks?.[uiLanguage] || []).map((fl, index) => (
```

Comment:
Things you realize while developing a new feature :laughing:

Reply:
Yeeeap 😅